concepts/05 UI Components/Gantt/20 Filter Data/2 Header Filter.md

```diff
@@ -0,0 +1,476 @@
+The **Gantt** component allows users to filter a particular column by values listed in a pop-up window. Users click the filter icon to invoke the window and select the necessary filter criteria for a column. 
```

Comment:
`the necessary filter criteria` Why do you add 'necessary'?

Reply:
I've used it as `what a user needs among the list of settings`. But if you think that use of this word is wordiness, we can remove it